homeassistant/components/vodafone_station/device_tracker.py

```diff
@@ -73,6 +73,11 @@ def __init__(
         self._attr_unique_id = mac
         self._attr_name = device.name or mac.replace(":", "_")
 
+    @property
+    def force_update(self) -> bool:
```

Comment:
After checking it with @joostlek, we conclude that `force_update` is already `False`. Please verify that this change is required.

Reply:
That's only for TrackerEntity, this is a ScannerEntity